frontend/src/components/ChatHistory/ChatHistoryListItem.tsx

```diff
@@ -258,12 +264,54 @@ export const ChatHistoryListItemGroups: React.FC<ChatHistoryListItemGroupsProps>
     );
   };
 
+    useEffect(() => {
```

Comment:
@sarah-widder Sorry I need another review here. I made this change because originally, when app loaded 25 records would be fetched. Then when the chat history panel was rendered another 25 would be fetched. I saw this as feature originally, but later decided against it as it felt clunky. These addition now makes sure **not** fetch on first render. This is the only change to the original PR.